koji-containerbuild.spec

```diff
@@ -54,6 +54,11 @@ Summary:    Hub plugin that extend Koji to build layered container images
 Group:      Applications/System
 Requires:   koji-containerbuild
 Requires:   koji-hub
+%if 0%{with python3}
```

Comment:
shouldn't this be under %package builder ?

Reply:
yes it should, fixed :)